docs/src/main/sphinx/security/catalog_rules.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
@mosabua Can we use these files as fragments in docs? Otherwise it is easy to get out of sync.

Reply:
You can import them or at least link to them.

Reply:
See https://github.com/trinodb/trino/blob/master/docs/src/main/sphinx/admin/resource-groups.rst

```
.. literalinclude:: resource-groups-example.json
    :language: json
```

Reply:
Thanks @mosabua  @kokosing 
Will make the change and get back.